api/managementpb/dbaas/components.proto

```diff
@@ -102,6 +102,43 @@ message ChangePXCComponentsRequest {
 
 message ChangePXCComponentsResponse {}
 
+// OperatorUpdateStatus represents status of an update for the operator.
+enum OperatorUpdateStatus {
+  OPERATOR_UPDATE_STATUS_INVALID = 0;
+  // An update of the operator is available for the installed version of PMM server.
+  UPDATE_AVAILABLE = 1;
+  // An update of the operator is not available for the installed version of PMM server.
+  UPDATE_NOT_AVAILABLE = 2;
+  // An update of the operator is available but for more up to date version of
+  // PMM server than is currently installed.
+  UPDATE_AVAILABLE_BUT_NOT_COMPATIBLE = 3;
+}
+
+message CheckForOperatorUpdateRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // operator_type tells what operator we are interested in for checking the update availability.
+  string operator_type = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message CheckForOperatorUpdateResponse {
+  OperatorUpdateStatus status = 1;
+  // available_operator_version is not empty if there is an update available,
+  // for both compatible and incompatible one.
+  string available_operator_version = 2;
+  // available_pmm_server_version is not empty only if the update is available but
+  // the version of the operator is not compatible with installed PMM version.
+  string available_pmm_server_version = 3;
```

Comment:
I think we should not return available operator version if it's not compatible with installed PMM version at all. And having this field makes no sense for me.

Reply:
We should be able to notify user that there is a new version available but he has to update PMM first.

Reply:
Removed.